types/eslint/use-at-your-own-risk.d.ts

```diff
@@ -0,0 +1,8 @@
+/** @deprecated */
+export const builtinRules: Map<string, import("./index.js").Rule.RuleModule>;
+/** @deprecated */
+export class FileEnumerator {
```

Comment:
these aren't technically deprecated - they're just "unstable" in that ESLint does not guarantee that they remain entirely stable between patch/minor versions.

Reply:
I can remove the deprecated tags, I suppose. I just figured a nice ~~strikethrough~~ in your editor would be a good marker that "you probably shouldn't be using this".